javascript/javascript3/week1/index.js

```diff
@@ -0,0 +1,56 @@
+
+// weahther check weather of city  
+
+        const btn=document.getElementById("btn")
+        const cityName=document.getElementById("cityName")
+        const outPut=document.getElementById("outPut")
+        let tagUi=document.createElement("ui");
+        outPut.appendChild(tagUi)
+        let tagli=document.createElement("li")
+        tagUi.appendChild(tagli);
+        tagli.style.listStyle="none";
+        let linebreak = document.createElement("br");
+
+                                //fucntion to convert timestamp to human readable
+          function Unix_timestamp(t)
+          {
+          var dt = new Date(t*1000);
+          var hr = dt.getHours();
+          var m = "0" + dt.getMinutes();
+          var s = "0" + dt.getSeconds();
+          return hr+ ':' + m.substr(-2) + ':' + s.substr(-2);  
+          }
+    
```

Comment:
Did you check if it actually works and shows time correctly? I'm afraid it doesn't.